### PR TITLE
Fix `releases-tab` and `bugs-tab`

### PR DESCRIPTION
**Pull request by cheap-glitch**
## Test URLs
Any repo page

## Screenshot
n/a

GitHub changed the layout of the navbar, now it's no longer a list but instead a simple collection of anchor elements. The anchors act both as links and as tabs, so they bear all the meta attributes directly.

### CODE REVIEW

File: source/features/bugs-tab.tsx [hunk 1]
```diff
@@ -87,7 +86,7 @@ async function init(): Promise<void | false> {
 	bugsCounter.title = '';
 
 	// Update Bugs’ link
-	new SearchQuery(bugsLink).add('label:bug');
+	new SearchQuery(bugsTab as HTMLAnchorElement).add('label:bug');
```
Comment (FloEdelmann) [inline source/features/bugs-tab.tsx:89]: Can you drop this type cast once you change the `issuesTab` selector to `a.UnderlineNav-item[data-hotkey="g i"]` (i.e. including the tag name `a`)?

Comment (cheap-glitch) [reply source/features/bugs-tab.tsx:89]: I thought so too, but unfortunately it seems that `cloneNode` loses the element type: https://github.com/Microsoft/TypeScript/issues/283

Comment (FloEdelmann) [reply source/features/bugs-tab.tsx:89]: Seems like the workaround added in #2670 doesn't work in this case?

Comment (cheap-glitch) [reply source/features/bugs-tab.tsx:89]: No it does actually, I just forgot the return value of `elementReady()` needed to be coerced. My bad.

**Review by fregante — APPROVED** [review]

File: source/features/bugs-tab.tsx (hunk 1 above)
Comment (fregante) [reply source/features/bugs-tab.tsx]: There’s an issue open on the element-ready repo to implement the selector parser for types, if anyone is interested. Relatively easy PR for that

**Comment by JourneyOver** [comment]
Looks like it's back to being a list once again xD

**Pull request merged**